includes/wikia/Wikia.php

```diff
@@ -2448,6 +2451,32 @@ public static function getCountryNames( array $countryCodes ) {
 	}
 
 	/**
+	 * Hide certain namespaces from list of movable ones
+	 *
+	 * @author pgroland
+	 * @see CE-1378
+	 *
+	 * @param index Index of namespace
+	 * @return bool
+	 */
+	static public function isNamespaceMovable( &$result, $index ) {
+		// namespace ids from extensions are hardcoded here
```

Comment:
Good point, Wikia.php is a global "utility class" loaded everywhere and not an extension.  I have seen we usually use the integer values when the extention is conditionally loaded (with a comment saying what it actually is like in this case). 

We could define a global $wgMovableNamespaceFilter that extensions could add to and this function could use.  Using globals "for great good" I suppose?